submissions/chabuChabu/javaScript/contactForm.js

```diff
@@ -0,0 +1,38 @@
+let submitForm = document.getElementById("contactForm"),
+    fullName = document.getElementById("fullName"),
+    email = document.getElementById("emailaddress"),
+    subject = document.getElementById("subject"),
+    textarea = document.getElementById("textarea")
+;
+
+function contactValidation(field,regex,errorOne,errorTwo){
+    event.preventDefault();
+    if(field.value === ""){
+        field.nextElementSibling.innerHTML = errorOne;
+        event.preventDefault();
+        return false;
+    }else if(!field.value.match(regex)){
+        field.nextElementSibling.innerHTML = errorTwo;
+        event.preventDefault();
+        return false;
+    }else{
+        field.nextElementSibling.innerHTML = "";
+    }
+}
+
+function validateContact(event) {
+  let fullNameRegex = /^([\w]{3,})+\s+([\w\s]{3,})+$/i,
+      emailRegex =  /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/,
+      subjectRegex = /^[A-Za-z]+$/,
+      messageRegex = /^[A-Za-z]+$/;
```

Comment:
`messageRegex` and `subjectRegex` are the same. Is there a reason it was declared twice?

Reply:
Comment addressed